Scripts/RemediationScripts/Remediate-SetSQLServerMinReqTLSVersion.ps1

```diff
@@ -0,0 +1,886 @@
+﻿<###
```

Comment:
Have proper line gap between different sections of Control Id and Display name etc. Kindly refer to other BRS

Reply:
Added gaps whereever required